static/src/scss/topic_list.css

```diff
@@ -46,3 +46,46 @@ html,
 body {
     height: 100%;
 }
+
+/* Spinner taken from https://github.com/tobiasahlin/SpinKit */
```

Comment:
What is this line suggesting?
Did you copy the spinner CSS from SpinKit? That code is licensed under MIT.
Let us install it with npm instead and load it with webpack.

Reply:
Caro took a very small code snippet and modified it. I think that adding a much bigger additional dependency and adding the modifying code on top would add unnecessary complexity rather than reducing it.

I would suggest to use the code as it is. If you like, I can remove the comment line with the reference.

Reply:
- Which part?
- How much modification?

Unnecessary complexity vs. reusing and modifying licensed software. This project is licensed under AGPL. The SpinKit is MIT. See
https://github.com/tobiasahlin/SpinKit/blob/master/LICENSE

I don't think removing the line is a good idea.